quickwit/quickwit-proto/src/indexing/mod.rs

```diff
@@ -153,6 +154,18 @@ impl ToString for IndexingTask {
     }
 }
 
+impl Eq for IndexingTask {}
+
+// TODO: This implementation conflicts with the default derived implementation. It would be better
```

Comment:
Why does it conflicts? It seems pretty normal?

Reply:
The default implementation takes `shard_ids` into account, whereas we only want to consider `index_uid` and `source_id`. I'll improve the comment.